src/rmq/rmqamqp/rmqamqp_connection.cpp

```diff
@@ -630,7 +630,9 @@ class Connection::ConnectionMethodProcessor {
         conn.sendConnectionCloseOk();
 
         if (closeMethod.classId() || closeMethod.methodId()) {
-            conn.d_retryHandler->errorCallback()("Connection error " +
+            conn.d_retryHandler->errorCallback()("Connection=" + 
+                                                     conn.d_connectionName +
+                                                     " Connection error " +
```

Comment:
```suggestion
                                                     " Error=" +
```

If it makes sense, otherwise lgtm

Reply:
I don't think we want to change this, we might have some alarms based on this. I believe it is always a connection error

Reply:
oh! we didnt do this as maybe we dont know if any one has any alarm based on that string

Reply:
haha. i should have refreshed.